lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ineligible.yaml

```diff
@@ -31,6 +39,22 @@ CRI_TICF_BEFORE_ANOTHER_WAY:
       targetJourney: TECHNICAL_ERROR
       targetState: ERROR_NO_TICF
 
+CRI_TICF_BEFORE_RETURN_TO_RP:
+  response:
+    type: process
+    lambda: call-ticf-cri
+  events:
+    next:
+      targetState: RETURN_TO_RP
+    enhanced-verification:
+      targetState: RETURN_TO_RP
```

Comment:
I think you're right. The only scenario that enhanced-verification could be triggered here is if the users is on a mitigation journey and opts to bail out from an escape route. Which means they'll have failed, and it will be more useful to them to see a page telling them that than just going back to the service. Will update.